sync-core/src/main/java/com/cloudant/mazha/CouchURIHelper.java

```diff
@@ -61,6 +63,17 @@ public URI changesUri(Map<String, Object> query) {
                 this.rootUriString,
                 "_changes"
         );
+
+        //lets find the since parameter
+        if(query.containsKey("since")){
```

Comment:
Tom might have a point, it's a slightly weird place to have this logic. But given the warped nature of some of this code it might be the most sensible right now. Who calls this method with a `since` param?